src/components/week/LoggedInUserWeekList.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import styled from "styled-components"
+import { listLikedContents } from "../../graphql/queries"
+import { API, graphqlOperation } from "aws-amplify"
+import { useQuery } from "react-query"
+import { ContentfulWeek } from "../../../graphql-types"
+import WeekCard from "./WeekCard"
+
+interface Props {
+  data: any
+}
+
+const LoggedInUserWeekList = (props: Props) => {
+  const fetchLikes = async () => {
+    return await API.graphql(graphqlOperation(listLikedContents))
+  }
+
+  const { data, status } = useQuery("someKeyName", fetchLikes)
+
+  return (
+    <Weeks>
+      {status === "loading" && <div>Loading data ...</div>}
```

Comment:
The error and loading states should be nice and never disable navigation (because we know that we have the data for the weeks, we just might not have data for the likes). Style these so that they display small loading spinner next to the title of the section or similar and if the query errors initialize either auto-retry or small nice-looking button to fetch the data again.